CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 * [ENHANCEMENT] Alertmanager: Add new additional template function `tenantID` returning id of the tenant owning the alert. #3758
 * [ENHANCEMENT] Reduce overhead of debug logging when filtered out. #3875
 * [ENHANCEMENT] Update Docker base images from `alpine:3.16.2` to `alpine:3.17.1`. #3898
+* [ENHANCEMENT] Update memcached image from `memcached:1.6.16-alpine` to `memcached:1.6.17-alpine`. #3914
```

Comment:
This should be part of Jsonnet section and Helm changelog. It's not an enhancement to Mimir itself.

Reply:
Thanks for the catch! I autopilot on the CHANGELOG sometimes.